frontend/src/pages/Overview/OverviewPage.tsx

```diff
@@ -1168,6 +1169,7 @@ const mapStateToProps = (state: KialiAppState): ReduxProps => ({
   refreshInterval: refreshIntervalSelector(state),
   minTLS: minTLSVersionSelector(state),
   istioAPIEnabled: state.statusState.istioEnvironment.istioAPIEnabled,
+  isMaistra: state.statusState.istioEnvironment.isMaistra,
```

Comment:
Take this opportunity to fix the alphabetization of these props.

Reply:
done